zulia-server/src/main/java/io/zulia/server/config/cluster/MongoNodeService.java

```diff
@@ -82,6 +84,11 @@ public void updateHeartbeat(String serverAddress, int servicePort) {
 
 	}
 
+	@Override
+	public void updateVersion(String version) {
+		getCollection().updateMany(new Document(), new Document("version", ZuliaVersion.getVersion()));
```

Comment:
This actually should be like updateHeartbeat and take a serverAddress and servicePort so it only updates this node's version

Reply:
Added.